### ISSUE TITLE
sorry very basic question

### ISSUE THREAD

**Issue by seeess1** [maxdee/alc_freeliner#52]
love freeliner. just wondering, do you have any advice on how to keep my laptop from crashing when i run freeliner? i'm working with a macbook air with 1.8 ghz and 8 gb of memory (so, nothing special, i know). it's been crashing after the first few clicks. anything i can modify to keep it running?

**Comment by maxdee** [comment]
Hello, glad you like freeliner! Please paste the console output here.

**Comment by seeess1** [comment]
Here are the error messages I'm getting. Still getting a computer shutdown
after I click a couple times and hit "n" to create a new item.

I think I'm doing everything you have listed in the instructions except I
haven't downloaded Syphon because I thought that was optional. Do you think
that might help prevent the shutdown?

**Comment by seeess1** [comment]
Hey just wanted to follow up to see if you got a chance to look at the
errors I was getting with freeliner?

**Comment by maxdee** [comment]
I did not get the log, please post it here on github.

**Comment by seeess1** [comment]
Oh sorry that's my mistake. Here are the error messages I'm getting:

![error_msg](https://user-images.githubusercontent.com/28066066/50922589-e4afae00-1418-11e9-9ac5-0cd5426ee147.jpeg)

![error_msg_continued](https://user-images.githubusercontent.com/28066066/50922590-e4afae00-1418-11e9-8b93-0a6e66694b84.jpeg)

**Comment by maxdee** [comment]
Can you run other Processing sketches? Try one of the opengl examples. I will let you know if I find anything else.

**Comment by seeess1** [comment]
Yep I'm able to run all the preloaded examples. And I've written a couple very basic animations that work fine.

**Comment by seeess1** [comment]
Mac OS 10.13

**Comment by seeess1** [comment]
Hey maxdee wanted to see if you had any ideas about how I can get freeliner to run on my computer

**Comment by maxdee** [comment]
I honestly have no idea.
Are you in NYC? I will be visiting mid February and could maybe meet up to debug it.

**Comment by seeess1** [comment]
Whoa that would be great. Yes I'm in NYC. Will be around for all of February. Happy to meet you whenever/wherever it's convenient

**Comment by seeess1** [comment]
By the way I found out about your project through the Recurse Center. I know a couple people who went through there. Such a cool place

**Comment by maxdee** [comment]
We observed that after making a few line segments, @seeess1 macbook air. Removing fixture layer stopped the problem. @seeess1 can you post your version of OSX and java?

**Comment by seeess1** [comment]
Yep I was running OS 10.13 and java 11.0.2
